### PR TITLE
MEW supports TomoChain Mainnet

### PR DESCRIPTION
**Pull request by thanhson1085**
Official announcement: https://medium.com/tomochain/tomochains-mainnet-launch-and-token-swapping-schedule-6f556e2f772

Network information: https://docs.tomochain.com/general/networks/

Thank MEW.

### CODE REVIEW

File: app/scripts/nodes.js [hunk 1]
```diff
@@ -267,16 +267,16 @@ nodes.nodeList = {
...
 	tomo: {
 		name: "TOMO",
-		blockExplorerTX: "https://scan.testnet.tomochain.com/txs/[[txHash]]",
-		blockExplorerAddr: "https://scan.testnet.tomochain.com/address/[[address]]",
+		blockExplorerTX: "https://scan.tomochain.com/txs/[[txHash]]",
+		blockExplorerAddr: "https://scan.tomochain.com/address/[[address]]",
 		type: nodes.nodeTypes.TOMO,
 		eip155: true,
-		chainId: 89,
+		chainId: 88,
```
Comment (gamalielhere) [inline app/scripts/nodes.js:274]: The chainID is conflicting with another. Your chainID is listed as 889 in https://github.com/satoshilabs/slips/blob/master/slip-0044.md

Comment (thanhson1085) [reply app/scripts/nodes.js:274]: As I know, slip-0044 is for registering HD path (m/44'/889'/0'/0)
I created the PR to slip44 (https://github.com/satoshilabs/slips/pull/365) and mentioned the chainID 88.
chainId 88 is also fixed in our genesis block

**Comment by thanhson1085** [comment]
This PR is very important with us. We already announced the mainnet launch.

I created the PR to slip44 (satoshilabs/slips#365) and mentioned the chainID 88.
chainId 88 is also fixed in our genesis block and not easy to change.

Please take a look @gamalielhere, thank you.

**Comment by Logle** [comment]
We are not aware of the practice using coin type index for chain ID for replay protection https://github.com/ethereum/EIPs/blob/master/EIPS/eip-155.md  There is no Ethereum fork with ChainID = 88. Please advise if we can use that.

**Review by gamalielhere — APPROVED** [review]

**Pull request merged**